en/identity-server/7.0.0/docs/apis/organization-apis/restapis/organization-management.yaml

```diff
@@ -322,8 +331,8 @@ paths:
                   },
                   "attributes": [
                     {
-                      "key": "Country",
-                      "value": "USA"
+                        "key": "Town",
```

Comment:
Is this indentation correct?

Reply:
Fixed with 90d8202096918adb80cd35941e9f4b3c4358e774